src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/memberdays/MemberDaysReport.java

```diff
@@ -138,15 +152,21 @@ private String getDatePart(String dateTimeValue) {
     }
 
     private String getHearingDuration(DateListedTypeItem dateListedTypeItem) {
-        var hearingStart = LocalDateTime.parse(dateListedTypeItem.getValue().getHearingTimingStart()
-            .replace(".000", ""));
-        var hearingFinish = LocalDateTime.parse(dateListedTypeItem.getValue().getHearingTimingFinish()
-            .replace(".000", ""));
-        var startFinishDuration = Duration.between(hearingStart, hearingFinish).toMinutes();
+        long startFinishDuration = 0;
+
+        if (dateListedTypeItem.getValue().getHearingTimingStart() != null
+            && dateListedTypeItem.getValue().getHearingTimingFinish() != null) {
+
+            var hearingStart = LocalDateTime.parse(dateListedTypeItem.getValue().getHearingTimingStart()
+                .replace(".000", ""));
```

Comment:
similar to previous comment, why are we using string manipulation rather than datetime formatter to parse date strings?

Reply:
Improved to use the datetime formatter. Please see the changes I just pushed.